### ISSUE TITLE
BMP and PNG images not loaded "correctly"

### ISSUE THREAD

**Issue by illume** [pygame/pygame#298]
Originally reported by: **Lillian Seabreeze (Bitbucket: [Lillian-Gardenia-Seabreeze](https://bitbucket.org/Lillian-Gardenia-Seabreeze), GitHub: Unknown)**

----------------------------------------

I made a simple image, which is a 2x3 image of red green blue, red green blue. When I load it, the colors are distorted, I can tell because I can use `get_at()` and the color is way off. If I save the image, too, the picture looks off. What's going on? This happens if I load a BMP or a PNG, and it doesn't matter if I use convert or not.

Running OS X (10.11.3). Pygame 1.9.2.dev1.

In summary, I'll open an image, I can test that the color data is wrong, and when I save it it's exactly as I expected when finding the color samplings wrong. Again, if pygame opens an image, the color data is super messed, and you can test such with `get_at()` and by saving said image.

----------------------------------------
- Bitbucket: https://bitbucket.org/pygame/pygame/issue/298
